docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -24,7 +24,7 @@ The tables in each of the tabs below document configuration properties that are
     |---|---|---|
     |Interval| 30s|The interval in seconds at which to report the metrics currently being collected and enabled. **Value of 0s disables reporting**. |
     |PublishTopicPrefix|"edgex/telemetry"|The base topic in which to publish (report) metrics currently being collected and enabled. `/<service-name>/<metric-name>` will be added to this base topic prefix.|
-    |Metrics|`<Service dependent>`|Boolean map of service metrics that are being collected. The boolean flag for each indicates if the metric is enabled for reporting. i.e. `EventsPersisted = true`. The metric name must match one defined by the service. |
+    |Metrics|SecuritySecretsRequested = false </br>SecuritySecretsStored = false</br> `<Service dependent>`|Boolean map of service metrics that are being collected. The boolean flag for each indicates if the metric is enabled for reporting. i.e. `EventsPersisted = true`. The metric name must match one defined by the service. |
```

Comment:
The HTML encoding is "<br/>" not "</br>"  This needs to change everywhere.

Reply:
actually, it should just be `<br>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
Fixed